prowler/providers/aws/services/s3/s3_service.py

```diff
@@ -378,6 +379,37 @@ def _get_bucket_tagging(self, bucket):
                     f"{error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
                 )
 
+    def _get_bucket_replication(self, bucket):
+        logger.info("S3 - Get buckets replication...")
+        try:
+            regional_client = self.regional_clients[bucket.region]
+            replication_config = regional_client.get_bucket_replication(
+                Bucket=bucket.name
+            )["ReplicationConfiguration"]["Rules"]
+            if replication_config:
+                bucket.replication = Rule(
+                    status=replication_config[0]["Status"],
```

Comment:
The buckets can have more than 1 replication rule, change the check accordingly please.